### DISCUSSION TITLE
osu!lazer usually crash after alt+tab or win+tab

### DISCUSSION THREAD

**Discussion by cuhoangdk** [ppy/osu#18170] [Q&A]
### Type

Crash to desktop

### Bug description

osu!lazer usually crash after alt+tab when i click something

### Version

Lazer 2022.501.0

### Logs

[runtime.log](https://github.com/ppy/osu/files/8646205/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8646206/updater.log)
[database.log](https://github.com/ppy/osu/files/8646207/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/8646208/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/8646209/network.log)
[performance.log](https://github.com/ppy/osu/files/8646210/performance.log)

**Comment by bdach** [comment]
The log does not contain a recognisable stack trace. Please attach Windows Event Viewer info by performing the following steps:
1. Type `Event viewer` in the Windows search bar (shortcut: <kbd>Windows</kbd>+<kbd>S</kbd>) and press <kbd>Enter</kbd>, or open the *Run...* button (<kbd>Windows</kbd>+<kbd>R</kbd>), type in `eventvwr` and press <kbd>Enter</kbd>.
2. The *Event viewer* window will appear. Expand *Windows Logs* on the left and choose *Application*. It might take a while to load.
3. In the *Actions* pane on the right click *Save All Events As...*, then choose a location to save to.
4. On the *Display Information* popup, select the second option, then the checkbox to show all available languages, then *English (UK)* (US would also do fine).
5. Append the resulting `.evtx` file to this issue.

**Reply by cuhoangdk** [reply]
Windows Event Viewer here
https://drive.google.com/file/d/1nQcA_kNq0hNQ8KwRd8KGVpAjW1vH6eGJ/view?usp=sharing

**Reply by peppy** [reply]
Same fault offset as https://github.com/ppy/osu/issues/17354.

Based on the other errors I'm seeing in that event viewer export, it seems like the issue may be disk/hardware level. There are many other applications falling over randomly.

**Reply by cuhoangdk** [reply]
for me it's just osu!lazer, other applications don't have this problem
